config/routes/mount_engine.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 Foreman::Application.routes.draw do
-  mount ForemanBootdisk::Engine, at: '/bootdisk'
+  mount ForemanBootdisk::Engine, at: '/', as: 'foreman_bootdisk'
```

Comment:
didn't test it yet, though would it break existing API calls or hard-coded routes to `/bootdisk/*`?

Reply:
If you change it to `/bootdisk` then test will fail:

```
be rails test:foreman_bootdisk

Failure:
ForemanBootdisk::Api::V2::DisksControllerTest::default API version 2#test_0001_path - /api/hosts/:host_id routes to #host [/home/lstejskal/projects/foreman_bootdisk/test/functional/foreman_bootdisk/api/v2/disks_controller_test.rb:81]
Minitest::Assertion: The generated path </bootdisk/api/v2/hosts/443> did not match </api/v2/hosts/443>.
Expected: "/api/v2/hosts/443"
  Actual: "/bootdisk/api/v2/hosts/443"

Failure:
ForemanBootdisk::Api::V2::DisksControllerTest::default API version 2#test_0002_path - /api/generic routes to #generic [/home/lstejskal/projects/foreman_bootdisk/test/functional/foreman_bootdisk/api/v2/disks_controller_test.rb:90]
Minitest::Assertion: The generated path </bootdisk/api/generic> did not match </api/generic>.
Expected: "/api/generic"
  Actual: "/bootdisk/api/generic"

66 tests, 151 assertions, 2 failures, 0 errors, 0 skips

```

Reply:
though it makes sense to me,
I saw this pattern across multiple plugins: https://github.com/search?q=org%3Atheforeman+engine_name&type=code
```rb
module ForemanPlugin
  class Engine < ::Rails::Engine
  engine_name 'foreman_plugin'
  ...
  ...
```

Reply:
here's also an example PR of how it was added in foreman_tasks: https://github.com/theforeman/foreman-tasks/commit/00cc6abfed25dcdba0ee56dbfe89de1d6af9128f